.travis.yml

```diff
@@ -1,8 +1,16 @@
-# Travis (http://travis-ci.org/) is a continuous integration
-# service for open source projects. This file configures
-# Travis to install and run "tox" test runner, which is
-# configured in tox.ini file.
-
+sudo: false
 language: python
-install: pip install tox --use-mirrors
-script: tox
+python: 3.5
```

Comment:
I would advice to specify all Python versions used here, and then use tox-travis to resolve the active Python version. Example: https://github.com/django-wiki/django-wiki/blob/master/tox.ini